robotic_arm:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_controller:
    type: arm_gazebo/MyPositionController
    joint: joint1
    gain: 100
    
  joint2_controller:
    type: arm_gazebo/MyPositionController
    joint: joint2
    gain: 100
    
  joint4_controller:
    type: arm_gazebo/MyPositionController
    joint: joint4
    gain: 100
    
  joint5_controller:
    type: arm_gazebo/MyPositionController
    joint: joint5
    gain: 100
    
  joint7_controller:
    type: arm_gazebo/MyPositionController
    joint: joint7
    gain: 100
